cloud/scope/object_storage_bucket.go

```diff
@@ -58,7 +62,7 @@ func NewObjectStorageBucketScope(ctx context.Context, apiKey string, params Obje
 		}
 		apiKey = string(data)
 	}
-	linodeClient, err := createLinodeClient(apiKey)
+	linodeClient, err := params.LinodeClientBuilder(apiKey)
```

Comment:
Couldn't we just call `CreateLinodeClient(apiKey)` since that returns a `*linodego.client` struct which satisfies `LinodeObjectStorageClient` interface?